test/server/meta.test.js

```diff
@@ -69,8 +69,27 @@ test('should fail to construct a script url with checkout.js on localhost withou
         error = err;
     }
 
+    if (error) {
+        throw new Error(`Should construct script with localhost url`);
+    }
+});
+
+test('should not construct a script url with checkout.js for non-supported local urls', () => {
+
+    const sdkUrl = 'http://not.a.supported.url:8000/api/checkout.js';
+
+    let error;
+
+    try {
+        unpackSDKMeta(Buffer.from(JSON.stringify({
+            url: sdkUrl
+        })).toString('base64'));
+    } catch (err) {
+        error = err;
+    }
+
     if (!error) {
-        throw new Error(`Expected error to be thrown`);
+        throw new Error(`Should construct script with supported local urls: (localhost, ngrok, loca)`);
```

Comment:
I think we removed `ngrok` as an option, right? Can we remove it here as well?
```suggestion
        throw new Error(`Should construct script with supported local urls: (localhost, loca)`);
```

Reply:
Yes. Thank you for the catch.  Don't want to confuse the requirements.